recipe/0118_multivalue/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
     "@context": "http://iiif.io/api/presentation/3/context.json",
-    "id": "https://example.org/iiif/text-language/manifest",
+    "id": "https://iiif.io/api/cookbook/recipe/0118_multivalue/manifest.json",
```

Comment:
Could you use "{{ id.url }}" as this should be filled with the correct location for both preview and iiif.io